src/styles/ant-vars.less

```diff
@@ -142,14 +142,14 @@ https://github.com/ant-design/ant-design/blob/master/components/style/themes/def
 @checkbox-group-item-margin-right: 8px;
 
 // Radio buttons
-@radio-button-bg: @black;
-@radio-button-checked-bg: @dark-three;
-@radio-button-color: @grayish-brown;
-@radio-button-hover-color: @primary-5;
-@radio-button-active-color: @dark-three;
-@radio-disabled-button-checked-bg: tint(@black, 90%);
-@radio-disabled-button-checked-color: @disabled-color;
-@radio-wrapper-margin-right: 8px;
+// @radio-button-bg: @black;
+// @radio-button-checked-bg: @dark-three;
+// @radio-button-color: @grayish-brown;
+// @radio-button-hover-color: @primary-5;
+// @radio-button-active-color: @dark-three;
+// @radio-disabled-button-checked-bg: tint(@black, 90%);
+// @radio-disabled-button-checked-color: @disabled-color;
+// @radio-wrapper-margin-right: 8px;
```

Comment:
might as well delete them

Reply:
Ok, will do. Makes it tempting to also delete all the other commented out rules...